### ISSUE TITLE
For training,how to organize the original Tusimple data set to get valid_index,json,gt_dir ....

### ISSUE THREAD

**Issue by HITjialanyu** [wvangansbeke/LaneDetection_End2End#7]

**Comment by wvangansbeke** [comment]
Hi, 

I use one directory for all the images on which you want to train (=data_dir) and one directory for all the ground truth data (=gt_dir).  TuSimple provides the user 3 json files with ground truth coordinates of the lane lines. I appended the three files and I used the index in the json files to name a specific image. For example: file "10.jpg" in data_dir corresponds with "10.png" in gt_dir and with index 10 in both label_data_all.json and Curve_parameters.json.

The original tuSimple data consists of 3626 images, however we used a subset of 2535 images. The dataloader will automatically assign 20% of the training folder to the validation set. I will put the files online to keep it easy for you. I will put a link in the README.md

Kind regards,
Wouter
